src/Listener/CodeCoverageListener.php

```diff
@@ -123,11 +121,9 @@ public function afterExample(ExampleEvent $event): void
      */
     public function afterSuite(SuiteEvent $event): void
     {
-        if (!$this->enabled || $this->skipCoverage) {
+        if ($this->skipCoverage) {
             if ($this->io && $this->io->isVerbose()) {
-                if (!$this->enabled) {
-                    $this->io->writeln('No code coverage will be generated as neither Xdebug nor phpdbg was detected.');
-                } elseif ($this->skipCoverage) {
+                if ($this->skipCoverage) {
```

Comment:
This check is rather unnecessary since we already know that this is true.

Reply:
Thank you for the review, I've updated the code and rebased on master.